packages/react-ui-core/src/List/List.js

```diff
@@ -71,9 +76,11 @@ export default class List extends PureComponent {
       >
         {items.map((item, i) => (
           <Item
+            {...itemProps}
+            highlight={!!(highlightIndex === i)}
```

Comment:
`highlightIndex === i` isn't already a bool? I'm guessing that's why you're doing `!!`

Reply:
Yeah, I added this for some reason before, I don't think we need it now.  It may have been for some test purpose.